_posts/2023-12-13-gift-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog_post
+title: "Compiler’s 2023 Holiday Gift Guide"
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered! Here is this year’s round-up of our favorite transit-related gift ideas.
```

Comment:
Copying from your message in Slack:

```suggestion
subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered! Here is this year’s round-up of our favorite transit-related gift ideas.
description: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered!"
```

Reply:
Oops, that was not the right place for me to insert that line... I fixed it in [539bcdf](https://github.com/compilerla/compiler.la/pull/150/commits/539bcdfea22d18fc60c67e82c0cdbed0e7229f7e) 😅